MiniSim/MenuItems/SubMenuItem.swift

```diff
@@ -145,26 +145,48 @@ enum SubMenuItems {
 }
 
 extension SubMenuItems {
-    static var android: [SubMenuItem] = [
+  static func items(platform: Platform, deviceType: DeviceType) -> [SubMenuItem] {
+    switch (platform, deviceType) {
+    case (.ios, .physical):
+      return [
+        CopyName(),
+        CopyUDID()
+      ]
+    case (.ios, .virtual):
+      return [
+        CopyName(),
+        CopyUDID(),
+
+        Separator(),
+
+        Delete()
+      ]
+    case (.android, .physical):
+      return [
         CopyName(),
         CopyID(),
 
         Separator(),
 
-        ColdBoot(),
-        NoAudio(),
         ToggleA11y(),
         Paste(),
-        DeleteEmulator(),
         LaunchLogCat()
-    ]
+      ]
```

Comment:
Some unnecessary diff

Reply:
Is it? I think the indentation changes because of the switch scope or?

Reply:
is it? the list is nested, so the bracket is indented. Does xcode remove the extra indentation for you when you format?

Reply:
Oh okay I missed that you added additional indentation. nevermind